brion/spikeReport.h

```diff
@@ -53,7 +53,7 @@ class SpikeReport;
  *
  * This class is not thread-safe except where stated otherwise.
  */
-class SpikeReport : public boost::noncopyable
+class SpikeReport
```

Comment:
Imo we removed it in other places in favor of the C++11 support for =delete. For readability one could put the private =delete section at the top of the class definition.